xds/internal/xdsclient/bootstrap/bootstrap_test.go

```diff
@@ -182,6 +182,22 @@ var (
 				}
 			]
 		}`,
+		"serverSupportsIgnoreResourceDeletion": `
```

Comment:
I think some of the tests need to include this filename `serverSupportsIgnoreResourceDeletion`. See how and where this `v3BootstrapFileMap` is used in different tests.